docs/using.md

```diff
@@ -538,13 +538,16 @@ Examples with these and other Actors can be found in [./src/workloads/docs](../s
 
 ### What is AutoRun?
```

Comment:
I think it would be nice to modify the header to include the deprecation as well:

```suggestion
### What was AutoRun?
```